server/routes/pinIPFS.ts

```diff
@@ -0,0 +1,19 @@
+import { Request, Response, NextFunction } from 'express';
+import { DB, sequelize } from '../database';
+import ipfs from '../util/ipfs';
+
+const pinIPFS = async (
+  models: DB,
+  req: Request,
+  res: Response,
+  next: NextFunction
+) => {
+  try {
```

Comment:
We need to validate the inputs here! 
Make sure the user is logged in (req.user), make sure we're unpacking whatever is being passed into the `req.body`, if we are verifying that it's a jsonblob or a string or... or if we're not validating, just make sure it's not empty.

Reply:
also validate that the `req.body.address_id` is not empty.

Reply:
Inputs are validated